### DISCUSSION TITLE
add some description text about directus project to header image

### DISCUSSION THREAD

**Discussion by sxsws** [directus/directus#13732] [Feature Requests]
directus is fairly high on this list: https://github.com/topics/no-code (found from duckduckgo search for no code builder)

There's no description shown so how about some simple text on top of the image like a lot of the other top projects?

**Comment by benhaynes** [comment]
Hmm, it's showing me "issues" first, but I don't see an option to show the "code" tab with the description. It must be automated somehow.

**Reply by benhaynes** [reply]
I submitted a topic heading... we'll see if it gets accepted.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
